lib/single-new-eks-opensource-observability-construct/graviton-index.ts

```diff
@@ -26,6 +26,9 @@ export default class SingleNewEksGravitonOpenSourceObservabilityConstruct {
         const nodeExporterDashUrl: string = utils.valueFromContext(scope, "nodeexporter.dashboard.url", undefined);
```

Comment:
This code is duplicated elsewhere, ideally we should centralize this case, create a data structure for all the input parameters, such as an interface for all URL, create a function that can populate this struct from the context and reuse it. 
This can be deferred, but if not addressed it will lead to more tech debt and maintenance overhead.

Reply:
Sounds good. We are working on this change with an active PR so deferring this for now. Agree on the maintenance overhead.